components/eventing-controller/cmd/eventing-controller/main.go

```diff
@@ -1,89 +1,77 @@
 package main
 
 import (
-	"context"
 	"flag"
 	"fmt"
 	"os"
+	"strings"
 	"time"
 
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/dynamic"
-	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	apigatewayv1alpha1 "github.com/kyma-incubator/api-gateway/api/v1alpha1"
-	eventingv1alpha1 "github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha1"
-	"github.com/kyma-project/kyma/components/eventing-controller/pkg/application"
-	"github.com/kyma-project/kyma/components/eventing-controller/pkg/env"
-	"github.com/kyma-project/kyma/components/eventing-controller/reconciler/subscription"
+	"github.com/kyma-project/kyma/components/eventing-controller/cmd/eventing-controller/beb"
+	"github.com/kyma-project/kyma/components/eventing-controller/cmd/eventing-controller/nats"
 )
 
-var scheme = runtime.NewScheme()
+const (
+	ENV_BACKEND = "BACKEND"
 
-func init() {
-	_ = clientgoscheme.AddToScheme(scheme)
-	_ = eventingv1alpha1.AddToScheme(scheme)
-	_ = apigatewayv1alpha1.AddToScheme(scheme)
+	ENV_BACKEND_VALUE_BEB  = "BEB"
+	ENV_BACKEND_VALUE_NATS = "NATS"
```

Comment:
Does it make sense to not export those constants outside of the main package?

Reply:
Yes, seen `env` package too late. Will change it.